src/dxtbx/model/detector.py

```diff
@@ -774,9 +774,11 @@ def imgCIF_H(cbf_handle, sensor):
         size = tuple(reversed(cbf_handle.get_image_size(0)))
 
         try:
-            underload = find_undefined_value(cbf_handle)
+            undefined_value = find_undefined_value(cbf_handle)
+            # In imgCIF overload means the pixel is saturated and hence untrusted
+            # https://www.iucr.org/__data/iucr/cifdic_html/2/cif_img.dic/Iarray_intensities.overload.html
             overload = cbf_handle.get_overload(0)
-            trusted_range = (underload, overload * dxtbx_overload_scale)
+            trusted_range = (undefined_value + 1, (overload - 1) * dxtbx_overload_scale)
```

Comment:
Here we rely _by convention_ on the fact that ["A value to be substituted for undefined values in the data array"](https://www.iucr.org/__data/iucr/cifdic_html/2/cif_img.dic/Iarray_intensities.undefined_value.html) is one less than the minimum trusted value of the detector. But actually I see nothing in the definitions that enforces that. `undefined_value` could indeed be anything, right?

In practice, we are always setting the minimum trusted value to zero, because `undefined_value` is apparently always -1. Perhaps we should skip the middleman and just explicitly set this to zero.

Reply:
Perhaps @phyy-nx might have thoughts on what the correct/desired behaviour here would be?

Reply:
Hi, my understanding is we should fall back to `undefined_value` if `underload` is not set, as proposed in #559.

Reply:
I was unaware of the `underload` field since I was looking at the version on the IUCr website, assuming that to be current. That begs the question why does the IUCr website only have the version 1.3.2 if there have been several more versions since then? It seems surprisingly hard to find the 1.8.4 version other than via your link in #559.

Reply:
Agree!  I don't know why it's so far behind!